test/bin/test-citgm.js

```diff
@@ -2,13 +2,17 @@
 
 var test = require('tap').test;
 
-var spawn = require('../../lib/spawn');
+var spawn = require('child_process').spawn;
```

Comment:
@bzoz @TheAlphaNerd what I don't quite understand is why `child_process.spawn` works on windows and `lib/spawn` doesn't what is the difference? It would be good to add windows support to `npm test`